generators/app/index.js

```diff
@@ -44,6 +44,14 @@ function repositoryPrefix(repo) {
   return (repo === 'terra-core' || repo === 'terra-framework') ? 'terra' : repo;
 }
```

Comment:
Thanks for adding doc to this, makes it a little easier to read which makes it a little easier to maintain.